physical_properties/pKa/example_submission_files/typeI-MehtapIsik-1.csv

```diff
@@ -0,0 +1,52 @@
+# Submission Type I
+#
+# PREDICTION SECTION
+#
+# Fill one typeI_microscopic_pKas_and_microstates.csv template for all molecules predicted with the same method.

```

Comment:
I understand what you are trying to say here, but its worded in a way that is a little confusing. Maybe something like:

You may submit predictions from multiple methods, but people make a separate [file name] for each different method. 
@davidlmobley help on wording?

Reply:
"Fill one typeI_microscopic_pKas_and_microstates.csv template for all molecules predicted with the same method."

Agree this is awkward. Maybe, "Each submission (method) must have a completed `typeI_microscopic_pKas_and_microstates.csv` file for all molecules the prediction encompasses," if that's what's meant?

Reply:
I liked how you clarified this rule. I added these sentences to pKa_challenge_instructions.md